Open Judge System/Data/OJS.Data/Repositories/SubmissionsForProcessingRepository.cs

```diff
@@ -25,14 +27,49 @@ public void AddOrUpdate(int submissionId)
             }
             else
             {
-                submissionForProcessing = new SubmissionForProcessing()
+                submissionForProcessing = new SubmissionForProcessing
                 {
                     SubmissionId = submissionId
                 };
+
                 this.Context.SubmissionsForProcessing.Add(submissionForProcessing);
             }
         }
 
+        public void BulkAddOrUpdate(IEnumerable<int> submissionIds)
```

Comment:
This is not a "Bulk" operation per se. Simply overload the `AddOrUpdate` method instead.